rest/model/task_annotations.go

```diff
@@ -0,0 +1,128 @@
+package model
+
+import (
+	"time"
+
+	"github.com/evergreen-ci/evergreen/model/task_annotations"
+)
+
+type APIAnnotationSource struct {
+	Author *string    `json:"author"`
+	Time   *time.Time `json:"time"`
+}
+type APIIssueLink struct {
+	URL      *string `json:"url"`
+	IssueKey *string `json:"issue_key"`
+}
+type APITaskAnnotation struct {
+	Id            *string             `json:"id"`
+	Note          *string             `json:"note"`
+	Issues        []APIIssueLink      `json:"issues"`
+	SuspectIssues []APIIssueLink      `json:"suspected_issues"`
+	Source        APIAnnotationSource `json:"source"`
+	Metadata      *string             `json:"metadata"`
```

Comment:
I recall a discussion in the context of the design about what type this field would be. I thought we came out it would be a json document.

Reply:
I set it to string so that I can codegen, I forgot to set it back to json. I'll switch it back.